.env.example

```diff
@@ -8,4 +8,6 @@ TABLE_NAME=test-table
 
 # PROJECT CONFIG
 OBJECTIVE=Solve world hunger
-FIRST_TASK=Develop a task list
\ No newline at end of file
+FIRST_TASK=Develop a task list
+
+USE_GPT4=false
```

Comment:
this is covered by a MODEL env var now, rather than a boolean, just choose from an enum